modules/local/count_matrix/combined/main.nf

```diff
@@ -0,0 +1,36 @@
+process COUNTS_COMBINED {
+    label 'process_low'
```

Comment:
no tag?

Reply:
nope, this is the step where multiple samples are used to make a count matrix, i.e multiple meta.id tags arriving